x-pack/plugins/lists/common/schemas/types/endpoint/entry_match_wildcard.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+
+import { NonEmptyString } from '../../../shared_imports';
+import { operatorIncluded } from '../../common/schemas';
+
+export const endpointEntryMatchWildcard = t.exact(
+  t.type({
+    field: NonEmptyString,
+    operator: operatorIncluded,
```

Comment:
As I imagine that this entry type will also end up being used in the detections side, do we want to constrain `operator` to only be `included` here? I can't remember how we handle that elsewhere, @marshallmain did we create separate entry types that have an operator of just `included` for endpoint?